.gitignore

```diff
@@ -12,6 +12,7 @@ MANIFEST
 .cache
 .pytest_cache
 .idea
+.yarn
```

Comment:
Note that this repo is currently using npm + package-lock.json, so please avoid adding yarn things.

Reply:
@vidartf Ok. I will change this.